sql/core/src/test/scala/org/apache/spark/sql/sources/v2/FileDataSourceV2FallBackSuite.scala

```diff
@@ -170,4 +174,46 @@ class FileDataSourceV2FallBackSuite extends QueryTest with SharedSQLContext {
       }
     }
   }
+
+  test("Fallback Parquet V2 to V1") {
```

Comment:
This test is flaky, it seems. Failed in my PR w/ unrelated changes:
```
sbt.ForkMain$ForkError: org.scalatest.exceptions.TestFailedException: ArrayBuffer(("collect", Relation [id#725095L] parquet
), ("command", InsertIntoHadoopFsRelationCommand file:/home/runner/work/spark/spark/target/tmp/spark-0d0623b6-2cdd-4ed0-bd86-0116ec152a01, false, Parquet, [path=/home/runner/work/spark/spark/target/tmp/spark-0d0623b6-2cdd-4ed0-bd86-0116ec152a01], ErrorIfExists, [id]
+- Range (0, 10, step=1, splits=Some(2))
)) had length 2 instead of expected length 1
	at org.scalatest.Assertions.newAssertionFailedException(Assertions.scala:472)
	at org.scalatest.Assertions.newAssertionFailedException$(Assertions.scala:471)
	at org.scalatest.Assertions$.newAssertionFailedException(Assertions.scala:1231)
	at org.scalatest.Assertions$AssertionsHelper.macroAssert(Assertions.scala:1295)
	at org.apache.spark.sql.connector.FileDataSourceV2FallBackSuite.$anonfun$new$22(FileDataSourceV2FallBackSuite.scala:179)
	at org.apache.spark.sql.connector.FileDataSourceV2FallBackSuite.$anonfun$new$22$adapted(FileDataSourceV2FallBackSuite.scala:175)
```
see https://github.com/MaxGekk/spark/runs/9650795751